bond_app/bond.py

```diff
@@ -77,6 +83,52 @@ def generate_access_token(self, sam_user_id):
                 "Could not find refresh token for sam_user_id: {} provider_name: {}\nConsider relinking your account to Bond.".format(
                     sam_user_id, self.provider_name))
 
+
+    def get_access_token(self, sam_user_id, refresh_threshold: int = 600):
+        """
+        Given a user, lookup their refresh token (if not provided) and use it to retrieve an access token from their OAuth
```

Comment:
```suggestion
        Given a user, lookup their refresh token and use it to retrieve an access token from their OAuth
```
No `refresh_token` parameter to this function.